Cargo.toml

```diff
@@ -34,6 +34,8 @@ windows = ["feat_os_windows"]
 nightly = []
 test_unimplemented = []
 expensive_tests = []
+# "test_bad_names" == enable tests that create problematic file names (would make a network share inaccessible to Windows, breaks SVN on Mac OS, etc.)
+test_bad_names = []
```

Comment:
bad is a bit vague ?
maybe test_invalid_file_name ?

Reply:
Well, they're not invalid on Unix, and can be fairly common in non-unicode encodings. They're just byte sequences that can cause problems with some tools that don't or can't handle them properly. Would you prefer "risky", or "dangerous"? (Also, if it wasn't clear, this will be used for `ls` tests as well.)

Reply:
yeah, risky then :)